server/controllers/feeds/lib/generate_feed_from_feed_data.js

```diff
@@ -2,30 +2,25 @@ const _ = require('builders/utils')
 const { feed: feedConfig } = require('config')
 const snapshot_ = require('controllers/items/lib/snapshot/snapshot')
 const serializeFeed = require('./serialize_feed')
-const getItemsByAccessLevel = require('controllers/items/lib/get_by_access_level')
 const getAuthorizedItems = require('controllers/items/lib/get_authorized_items')
 const user_ = require('controllers/user/lib/user')
+const { filterPrivateAttributes } = require('controllers/items/lib/filter_private_attributes')
+const { paginate } = require('controllers/items/lib/queries_commons')
 
-module.exports = lang => async ({ accessLevel, reqUserId, feedOptions, users, shelves }) => {
+module.exports = lang => async ({ reqUserId, feedOptions, users, shelves, filter }) => {
```

Comment:
could we document what `filter` could mean in this context ?

Reply:
Maybe `filter` is the wrong name here: what about renaming that key to `context`, which then lets the `paginate` function determine if a filter should be applied? Currently, the only use-case is when items from a group are requested, and that in that context, the requested private items are filtered out to not give the false impression that those private items appear in the group

Reply:
it does filter though, renaming is not necessary to me, a simple comment similar to the one in `get_by_users.test.js`
```suggestion
// filter known case: reject private items in group context to not give the false impression that those are visible to the other group members 
module.exports = lang => async ({ reqUserId, feedOptions, users, shelves, filter }) => {
```

Reply:
done in 61ebcbda9